packages/testwatch/index.js

```diff
@@ -220,7 +220,7 @@ ${Object.entries(this.#currentCommands)
   }
 
   async run() {
-    await this.#runTests();
+    this.#runTests();
```

Comment:
why?

Reply:
If we await the `runTests` function, we don't catch the `drained` event we fire where no file is found.